website/docs: Flesh out integrations copy changes. (#13619)
goauthentik/authentik: website/integrations/index.mdx

---
title: Integrations overview
sidebar_label: Overview
---

## What is an integration?

An integration is a how authentik connects to third-party applications, directories, and other identity providers.
Integrations are categorized into two categories: **Applications** and **Sources**.

### Applications

> Applications include vendor tools such as Google Workspace, GitHub, Slack, or AWS. These applications can be integrated with authentik to provide single sign-on capabilities to

Reviewer: ```suggestion
Applications include vendor tools such as Google Workspace, GitHub, Slack, or AWS. These applications can be integrated with authentik to provide single sign-on capabilities to securely authenticate users.
```

> If you're looking to integrate an application that isn't listed, authentik can be configured to work with most applications that support authentication protocols such as [SAML](../docs/add-secure-apps/providers/saml/index.md), [OAuth and OpenID Connect](../docs/add-secure-apps/providers/oauth2/index.mdx).

Reviewer: ```suggestion
If you want to integrate an application that isn't listed, authentik can be configured to work with most applications that support authentication protocols such as [SAML](../docs/add-secure-apps/providers/saml/index.md), [OAuth and OpenID Connect](../docs/add-secure-apps/providers/oauth2/index.mdx).
```

To learn more, refer to the [Applications](../integrations/services/index.mdx) page.

> 

Reviewer: ```suggestion
```

Author: Newline is being added by the formatter.

### Federated and social sources

> Sources are a way for authentik to use external credentials for authentication, authentik also integrates with external sources, including federated directories like Active Directory and through protocols such as [LDAP](../docs/users-sources/sources/protocols/ldap/index.md), [SCIM](../docs/users-sources/sources/protocols/scim/index.md), [SAML](../docs/users-sources/sources/protocols/saml/index.md), and [OAuth and OpenID Connect](../docs/users-sources/sources/protocols/oauth/index.mdx)

Reviewer: ```suggestion
Sources are a way for authentik to use external user credentials for authentication. Supported integrations with external sources via authentik include federated directories like Active Directory and social logins such as Facebook, Twitter, etc. These integrations support all major protocols, including [LDAP](../docs/users-sources/sources/protocols/ldap/index.md), [SCIM](../docs/users-sources/sources/protocols/scim/index.md), [SAML](../docs/users-sources/sources/protocols/saml/index.md), and [OAuth and OpenID Connect](../docs/users-sources/sources/protocols/oauth/index.mdx)
```

> 

Reviewer: ```suggestion
```

> Sources in authentik can also be used for social logins, using external providers such as Facebook, Twitter, etc.

Reviewer: ```suggestion
```
